index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Microverse Portfolio</title>
+</head>
+<body>
+    <section class="header">
+        <nav>
+            <p>My Logo</p>
+            <img src="./assets/menu.png" alt="logo">
+        </nav>
```

Comment:
- [ ] Please wrap the `nav` with a `header` tag so it can look like this:

```javascript
<header>
   <nav>
      ...
   </nav>
</header>
```

Reply:
Please I want to know if I should add the `header` tag inside my section tag